tutorial/example1.py

```diff
@@ -1,6 +1,6 @@
 import os
-import reframe.utility.sanity as sn
 
+import reframe.utility.sanity as sn
```

Comment:
You should update also the `.rst` documentation files accordingly. Perhaps, it's better to use [this](https://goo.gl/u2mD9k) and avoid synchronizing the tutorial examples and the documentation every time. If it turns out not to be straightforward, you can just update the `.rst` files manually and we postpone this item.

Reply:
Done.